### DISCUSSION TITLE
023

### DISCUSSION THREAD

**Discussion by giscus** [zhiyu1998/rrorange-and-friends-weekly#12] [General]
# 023

《大橘和朋友们的周刊》： 分享日常冲浪互联网看到好玩的网站、app应用、资源分享、效率软件工具集等

https://rrorangeandfriends.site/posts/2024/023

**Comment by shannan77** [comment]
huang111网盘推荐的太晚了，已经关闭了

**Reply by zhiyu1998** [reply]
那太可惜了，我也是最近才发现的，因为下载特别丝滑所以才推荐出来

**Comment by Leridy** [comment]
看到我做的划水网了。服务器性能羸弱，暂未开发自动爬取所有网站的功能。数据信息全靠用户访问来触发。所以打开页面之后全是空的。
今天（2025/03/02）我又更新了一些新的彩蛋功能。欢迎访问
